howto/use-w3c-tracecontext/README.md

```diff
@@ -0,0 +1,243 @@
+ * providing an industry standard that intermediaries, platforms, and hardware providers can support.
+
+A unified approach for propagating trace data improves visibility into the behavior of distributed applications, facilitating problem and performance analysis. The interoperability provided by trace context is a prerequisite to manage modern micro-service based applications.
+
+Please refer complete W3C context specifications [here](https://www.w3.org/TR/trace-context/).
+
+## W3C Trace Headers
+
+### Trace Context HTTP Headers Format
+
+#### Traceparent Header
+
+The traceparent header represents the incoming request in a tracing system in a common format, understood by all vendors. 
+Here’s an example of a traceparent header.
+
+`traceparent: 00-0af7651916cd43dd8448eb211c80319c-b7ad6b7169203331-01`
+
+Please refer traceparent fields details [here](https://www.w3.org/TR/trace-context/#traceparent-header)
+
+#### Tracestate Header
+
+The tracestate header includes the parent in a potentially vendor-specific format:
+
+`tracestate: congo=t61rcWkgMzE`
+
+Please refer tracestate fields details [here](https://www.w3.org/TR/trace-context/#tracestate-header)
+
+### Trace Context gRPC Headers Format
+
+In the gRPC API calls, trace context is passed through `grpc-trace-bin` header.
+
+
+## Using Trace Context in Dapr
+
+Dapr tracing is built on  [OpenCensus](https://opencensus.io/introduction/) specifications that supports official W3C HTTP tracing header.
+For the gRPC tracing , [here](https://github.com/census-instrumentation/opencensus-specs/blob/master/trace/gRPC.md) is more details with OpenCensus.
+
+Dapr provides in built support of OpenCensus instrmentation of services when tracing configuration is enabled through Dapr annotation.
+Once the tracing configuration is applied, you will get full instrumentation of traces and you can retrieve the trace correlation id from the standard
+W3C Context headers as outlined earlier.
+
+However if you choose to pass the trace context explictly, then Dapr will use the passed trace context and propagate all across the HTTP/gRPC call.
+
+## How to pass Trace Context
+
+Since Dapr tracing is built on OpenCensus, user needs to send trace context using OpenCensus SDK.
+OpenCensus supports majority of languages.
+
+| Language | SDK |
+|:-------:|:----:|
+| Go | [Link](https://pkg.go.dev/go.opencensus.io?tab=overview)
+| Java | [Link](https://www.javadoc.io/doc/io.opencensus/opencensus-api/latest/index.html)
+| C# | [Link](https://github.com/census-instrumentation/opencensus-csharp/)
+| C++ | [Link](https://github.com/census-instrumentation/opencensus-cpp)
+| Node.js | [Link](https://github.com/census-instrumentation/opencensus-node)
+| Python | [Link](https://census-instrumentation.github.io/opencensus-python/trace/api/index.html)
+
+For the purpose of example, we will use the same [grpc app](../create-grpc-app) based on Go SDK.
```

Comment:
Please rephrase to remove all first person references ("we", "I")

Reply:
done.